bluemira/equilibria/profiles.py

```diff
@@ -639,6 +642,18 @@ def parse_to_callable(unknown):
         else:
             raise TypeError("Could not make input object a callable function.")
 
+    def pprime(self, pn):
+        """
+        dp/dpsi as a function of normalised psi
+        """
+        return self.scale * self._pprime(pn)
```

Comment:
ref @alexanderianblair for the scaling

Reply:
#563 . We should merge that in first otherwise there will be conflicts and this PR is not related to this issue

Reply:
Please merge that as soon as you can so we can get this merge completed, along with the one on the STEP repo. The longer we wait the more likely the latter-most will need to have more conflicts resolved.